package.xml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0"?>
-<package>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
   <name>steering_functions</name>
   <version>0.2.0</version>
```

Comment:
Shall we increment the version?

Reply:
We can change the version number, but it is not needed. There was no API break.

Reply:
When this software is released with bloom on a later date, the patch number will be incremented

Reply:
OK